### ISSUE TITLE
Create RetroPie install option

### ISSUE THREAD

**Issue by martinohanlon** [martinohanlon/mayhem-pi#29]
So it can be installed from the RetroPie setup menu

**Comment by martinohanlon** [comment]
Create PR https://github.com/RetroPie/RetroPie-Setup/pull/1828

**Comment by ghost** [comment]
I've just updated my script but I have issues when starting the game. The start.sh script cannot get the RAM info requested unless it's run by root:

pi@retropie:/opt/retropie/ports/mayhem $ ./start.sh
./start.sh: line 4: gpu_mem: Permission denied
./start.sh: line 5: gpu_mem: No such file or directory
./start.sh: line 8: ((: >= 96 : syntax error: operand expected (error token is ">= 96 ")
Mayhem 2 needs at least 96MB of memory allocated to the GPU
Use sudo raspi-config (Advanced Options > Memory Split) to change
rm: cannot remove ‘gpu_mem’: No such file or directory
pi@retropie:/opt/retropie/ports/mayhem $

When running mayhem2-pi:

pi@retropie:/opt/retropie/ports/mayhem $ ./mayhem2-pi
Allegro started version 5.0.10[1]
Found 0 joysticks
mayhem2-pi: /build/allegro5-VVCIRi/allegro5-5.0.10/src/events.c:150: al_register_event_source: Assertion `source' failed.
Aborted
pi@retropie:/opt/retropie/ports/mayhem $

Any help would be appreciated.

**Comment by martinohanlon** [comment]
The version of allegro packaged is old v5.0.10 and it is pre changes to make it stable and performant on the Raspberry Pi, which I think came in version 5.1.something - you really want 5.2 though.

**Comment by ghost** [comment]
Yeah, I don't see that in jessie. Also, installing pulseaudio seems to make emulationstation go berzerk, taking up 100% CPU at all times.
